external/iotjs/src/platform/tizenrt/iotjs_main-tizenrt.c

```diff
@@ -116,9 +116,11 @@
  pthread_addr_t iotjs_thread(void *thread_arg) {
    struct iotjs_thread_arg *arg = thread_arg;
 
-   iotjs_entry(arg->argc, arg->argv);
+   int ret = iotjs_entry(arg->argc, arg->argv);
```

Comment:
We use -Werror option. When CONFIG_DEBUG is disabled, this variable causes compilation error.
"variable is defined but not used".

Reply:
And this is different kind of changes. Let's split commits.